src/main.cc

```diff
@@ -61,6 +63,7 @@ R"(Written by Andrea Barberio - https://insomniac.slackware.it
   -T MAX_TTL --max-ttl=MAX_TTL  the maximum TTL to probe. Must be greater or equal than the minimum TTL (default: )" << static_cast<int>(DublinTraceroute::default_max_ttl) << R"()
   -D DELAY --delay=DELAY        the inter-packet delay in milliseconds (default: )" << DublinTraceroute::default_delay << R"()
   -b --broken-nat               the network has a broken NAT configuration (e.g. no payload fixup). Try this if you see less hops than expected
+  -i --iterate-sport            iterate the source port instead of the destination port
```

Comment:
For clarity can you change this to "Generate paths using source port instead of destination port" ?

Reply:
Fixed!